armi/utils/tests/test_utils.py

```diff
@@ -24,26 +24,47 @@
 from armi.settings.caseSettings import Settings
 from armi.tests import mockRunLogs
 from armi.utils import (
+    codeTiming,
     directoryChangers,
-    getPowerFractions,
-    getCycleNames,
     getAvailabilityFactors,
-    getStepLengths,
-    getCycleLengths,
     getBurnSteps,
+    getCumulativeNodeNum,
+    getCycleLengths,
+    getCycleNames,
+    getCycleNodeFromCumulativeNode,
+    getCycleNodeFromCumulativeStep,
+    getFileSHA1Hash,
     getMaxBurnSteps,
     getNodesPerCycle,
-    getCycleNodeFromCumulativeStep,
-    getCycleNodeFromCumulativeNode,
+    getPowerFractions,
     getPreviousTimeNode,
-    getCumulativeNodeNum,
+    getStepLengths,
     hasBurnup,
-    codeTiming,
     safeCopy,
 )
 
 
 class TestGeneralUtils(unittest.TestCase):
+    def test_getFileSHA1Hash(self):
+        with directoryChangers.TemporaryDirectoryChanger():
+            path = "test.txt"
+            with open(path, "w") as f1:
+                f1.write("test")
+            sha = getFileSHA1Hash(path)
+            self.assertIn("a94a8", sha)
```

Comment:
I'm trying to decide if this test is fully stable.

I mean, it passed GitHub CI once, so maybe. Probably?

Different operating systems, different implementations of SHA1, different timestamps.  

I guess this works. Huh.

Reply:
I guess, as long as the contents of the file are always utf-8, the hash will always be the same.

Okay, sure.

Reply:
Yea I was actually trying to figure out how to test this and TOTALLY thought it would be unstable. So I was nervous about this test.

But it passed in 2 different places and then also the CI here so I guess it must be!